### PR TITLE
Engineering questions

### PR DESCRIPTION
**Pull request by ncantor**
Shortened the job spec.
The goal was to present the job spec as an invitation to a conversation.
Explicitly stated in the README, now.
Initial list of technologies currently being used in the engineering team.
Removed anything which didn't add to the goal of painting a picture / inviting a conversation.

### CODE REVIEW

File: Recruitment/JobSpecs/why_would_you_join_us.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+Do you want to help them figure out how to think and work differently?
+
+Do you love technology, and understand that real change comes from people doing different things, rather than people doing the same things differently?
+
+These are the things that drive us. Do they drive you?
+
+### Who we are looking for
+
+We are looking for people who want to help us change the way companies think and approach problems; people who want to work with teams and individuals who share a passion for solving complex problems; people who want to own the entire problem, whether it’s about technology, ways of working, teams, or poor customer experience.
```
Comment (davidmoss) [inline Recruitment/JobSpecs/why_would_you_join_us.md]: How about changing "poor customer experience" -> "improving customer experience through fast and continuous iterations"

Comment (ncantor) [inline Recruitment/JobSpecs/why_would_you_join_us.md]: Seems wordy to me. What would be the benefit of changing it, do you think?

Comment (panosvoudouris) [inline Recruitment/JobSpecs/why_would_you_join_us.md]: "poor customer experience" ends the sentence in a negative. The alternative suggested is more positive and also gives another hint into how we like to work. Could be shortened to just "improving customer experience" if we don't want something that long.

Comment (parrymandal) [inline Recruitment/JobSpecs/why_would_you_join_us.md]: why qualify the customer experience part and not the others? so is either improving or poor needed?

**Pull request merged**